lib/ronin/web/cli/commands/diff.rb

```diff
@@ -100,6 +107,27 @@ def read(source)
             end
           end
 
+          private
+
+          def load_doc(page1, page2)
+            case options[:format].to_sym
+            when :html
+              [load_html(page1), load_html(page2)]
+            when :xml
+              [load_xml(page1), load_xml(page2)]
+            else
+              raise "Unsupported format"
+            end
+          end
+
+          def load_html(page)
```

Comment:
also needs documentations, or could be merged into `load_doc`? I will let you decide.

Reply:
applied suggestion